src/.editorconfig

```diff
@@ -0,0 +1,207 @@
+# Remove the line below if you want to inherit .editorconfig settings from higher directories
```

Comment:
># [](start = 0, length = 1)

I you tell me you copied this from the SARIF SDK then I won't review it line by line. I reviewed the SARIF SDK's .editorconfig file when it was first added.